packages/next-ui/Form/FormLayout.tsx

```diff
@@ -0,0 +1,19 @@
+import { Box, SxProps, Theme } from '@mui/material'
+
+export type UseFormLayoutProps<T> = Pick<FormLayoutProps<T>, 'children'>
+
+export type FormLayoutProps<T> = {
+  form: T
+  original: React.ReactNode
+  children?: (
+    props: {
+      original: React.ReactNode
+    } & T,
+  ) => React.ReactNode
+  sx?: SxProps<Theme>
+}
+
+export function FormLayout<T>(props: FormLayoutProps<T>) {
+  const { children, original, form, sx } = props
+  return <Box sx={sx}>{children ? children({ original, ...form }) : original}</Box>
```

Comment:
Wouldn't it be better to use a fragment instead of a Box so we don't introduce unnecessary div's.

Reply:
This probably is a bit much. It's wrapped in a fragment now 👍